damus/Views/Profile/ProfileView.swift

```diff
@@ -127,6 +154,8 @@ struct ProfileView: View {
                             .clipped()
 
                         VisualEffectView(effect: UIBlurEffect(style: .systemUltraThinMaterial)).opacity(bannerBlurViewOpacity())
+                        
+                        bannerTextView(minY: minY)
```

Comment:
We should put the banner text view in a different place where the layout engine can automatically position the element properly.

Currently this strategy only looks good on iPhone 16 Pro in landscape mode, or devices with similar screen size — but looks off on other configurations/screens. Examples:

iPhone SE:
<img width="811" alt="Screenshot 2024-12-03 at 5 39 47 PM" src="https://github.com/user-attachments/assets/13f99ddd-37bd-4a1b-80cb-b4cbcd35e56d">

iPhone 16 in landscape mode:
<img width="824" alt="Screenshot 2024-12-03 at 5 35 21 PM" src="https://github.com/user-attachments/assets/660138b9-e91a-4baf-b851-30038ab42ae1">

Perhaps this should be a `ToolbarItem`?